tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/GacCommandTests.cs

```diff
@@ -51,24 +52,26 @@ public void GacCompleteTest()
         // We skip the test if we don't have permissions
         Skip.IfNot(Gac.AdministratorHelper.IsElevated);
 
-        var assemblyName = "DummyLibrary";
-        var assemblyPath = Path.GetDirectoryName(typeof(GacCommandTests).Assembly.Location);
-        assemblyPath = Path.GetFullPath(Path.Combine(assemblyPath, assemblyName + ".dll"));
+        const string DummyAssemblyName = "DummyLibrary";
+        var rootFolder = Path.GetDirectoryName(typeof(GacCommandTests).Assembly.Location)!;
+        var assemblyPathV1 = Path.GetFullPath(Path.Combine(rootFolder, $"{DummyAssemblyName}V1.dll"));
+        var assemblyPathTarget = Path.GetFullPath(Path.Combine(rootFolder, $"{DummyAssemblyName}.dll"));
```

Comment:
I wonder if it would be better to use the exact same assembly name (and put them in `V1`/`V2` sub folders) instead of renaming, to more closely emulate the potential issues we could hit. I don't know that the file path _matters_ for this, but maybe? Does it affect the assembly simplename?